README.md

```diff
@@ -113,6 +117,8 @@ If there are no errors you should see the following
 > tsc
 ```
 
+NOTE: If the above setting contains different account/region than initial `cdk bootstrap`, will then need to perfrorm `cdk bootstrap` again to avoid error. Please reference [CDK](https://docs.aws.amazon.com/cdk/latest/guide/home.html) usage doc for detail.
```

Comment:
Note: if the account/region combination used in the code example above is different from the initial combination used with `cdk bootstrap`, you will need to perform `cdk bootstrap` again to avoid errors. 
Please reference [CDK](https://docs.aws.amazon.com/cdk/latest/guide/home.html) usage doc for details.